FordLabs/PeopleMover
# [#226] Disallow assignments to inactive (archived or ended) products

## Issue
Resolves 226 disallow assignments to inactive (archived or ended) products

## What was done
- [x] Add check to code to filter both archived and ended products from the Assign To dropdown list on PersonForm

## How to test
1. In a space with ended or archived products, open a person to edit them. 
2. Note that the products that are archived or ended do not show up in the Assign To dropdown.
